storybookjs/aem
# fix(app): add logic for component inheritance

Issue:
relates to #45

## What I did
- added logic to component loader to resolve the script based on `sling:resourceSuperType`
- import some of the aem-core-components in the aem-kitchen-sink and created extended component (AEM Text).
- improved webpack component loader to better detect the resource type
-moved some of the aem-core-components below `jcr_root` so that the resource type can be detected

## How to test
- build and run the aem-kitchensink-example
- there should be a functional `AEM Text` story

## Still TODO:
- improve the way to determine the resource type
- find a way to add multiple template roots to HTL engine
